RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -68,7 +71,7 @@ void HLTPFDiJetCorrCheckerWithDiTau::produce(edm::StreamID iSId, edm::Event& iEv
 
   std::set<unsigned int> indices;
 
-  if (pfJets.size() > 1 && taus.size() > 1) {
+  if (pfJets.size() > 1 && taus.size() > 0) {
```

Comment:
Since the loop is on jet pairs, this `if` statement looks redundant altogether. Doesn't the plugin behave the same without it? If so, please remove it.

Reply:
"As far as I understand, this is because that single use case in the current HLT is preceded by a filter requiring the input taus are at least 2. In general, this update changes the behaviour of the filter. Is that correct?" 
Correct. The behavior of the filter is the same in the case of 2 jets and 2 taus and the proposed change allows the same cross-cleaning/correlation-checking to be performed with 2 jets and 1 tau.

The plugin behaves the same without the if statement, as you pointed out. It has been removed